backend/src/main/java/de/uslu/Travel/TravelService.java

```diff
@@ -0,0 +1,22 @@
+package de.uslu.Travel;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.Collection;
+
+@Service
+@RequiredArgsConstructor
+public class TravelService {
+
+    private final TravelRepo travelRepo;
+
+    public Collection<Travel> listTravels(String travel) {
+        return travelRepo.findAllByCity(travel);
+    }
+
+    public Travel createTravel(Travel travel, String city) {
+         travel.setCity(city);
+         return travelRepo.save(travel);
```

Comment:
```suggestion
    public Travel createTravel(Travel travel) {
         return travelRepo.save(travel);
```

Here you the the Username as city. Controller sends travelItem and principal.getName. Pretty sure the Principal is not a city. Either use a Pathvariable or send the City with your travelDTO.